clang/test/Frontend/sycl-unnamed-lambdas-default.cpp

```diff
@@ -1,18 +1,14 @@
 // RUN: %clang_cc1 -fsycl-is-device %s -verify -DUNNAMED_LAMBDAS
 // RUN: %clang_cc1 -fsycl-is-host %s -verify -DUNNAMED_LAMBDAS
 //
-// RUN: %clang_cc1 -fsycl-is-host -sycl-std=2017 %s -verify
+// RUN: %clang_cc1 -fsycl-is-host -sycl-std=2020 %s -verify
```

Comment:
I think that we can remove this `RUN` line, because it doesn't really check anything.